molecule/default/molecule.yml

```diff
@@ -33,11 +33,24 @@ platforms:
       - name: cartridge-network
     purge_networks: true
     privileged: true
+  - name: vm3
+    image: centos:7
+    command: /sbin/init
+    tmpfs:
+      - /run
+      - /tmp
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    published_ports:
+      - 8071-8080:8071-8080/tcp
+    networks:
+      - name: cartridge-network
+    purge_networks: true
 
 lint: |
   set -xe
   yamllint .
-  flake8 library molecule/default/tests
+  flake8 library molecule/resources/tests
```

Comment:
Please, move tests back to scenario folder. Now tests aren't ran on `molecule test`.
Moreover, you need to write additional tests for TGZ.